wuvt/defaults.py

```diff
@@ -30,7 +30,7 @@
 
 # DJ activity timer for automation/DJ logout in seconds
 DJ_TIMEOUT = 30 * 60
-EXTENDED_DJ_TIMEOUT = 120 * 60
+EXTENDED_DJ_TIMEOUT = 240 * 60
```

Comment:
Some DJs have had problems with this when doing live shows. I can move this to a separate PR if you prefer.